packages/rich-text/src/to-dom.js

```diff
@@ -61,6 +61,10 @@ function append( element, child ) {
 		child = element.ownerDocument.createTextNode( child );
 	}
 
+	if ( child.html !== undefined ) {
+		return ( element.innerHTML += child.html );
+	}
+
```

Comment:
Does this need to be documented and perhaps mentioned in the change log?

Reply:
It's not an external API